core/templates/expressions/expression-syntax-tree.service.ts

```diff
@@ -35,7 +35,8 @@ interface EnvDict {
 }
 
 export class ExpressionError extends Error {
-  // 'message' is optional beacuse we may not want a custom error message.
+  // 'message' is optional beacuse it is optional is the actual 'Error'
```

Comment:
This, as written, does not make sense. Please clarify it.

Reply:
updated! is it fine now?

Reply:
Yes, much better -- thanks!